performance/performance.md

```diff
@@ -1,95 +1,156 @@
 
 ###### source: [scalewp.io](https://www.scalewp.io/object-caching/)
 
-What does this mean? Think of a standard WordPress homepage displaying the most recent posts. Each of these posts has quite a bit of information associated with it WordPress must look up such as the author, categories, tags, excerpt, etc. Out of the box WordPress will throw this information away after each request and fetch it again for the next visitor to the site.
+What does this mean? Think of a standard WordPress homepage displaying the most recent posts. Each of these posts has quite a bit of information associated with it WordPress must look up such as the author, categories, tags, and excerpt.
 
-A persistent object cache gives WordPress a place to store data for re-use. Like other forms of caching this data will expire after a given amount of time. While these objects are cached, however, PHP execution time is improved while lessening the load on the database.
+Support for a persistent object cache gives WordPress, plugins, and themes, a place to store that data for reuse. While these items are cached, PHP execution time is improved while lessening the load on the database. It's particularly helpful in situations where much of the page is difficult to cache from the front-end, like for authenticated traffic or e-commerce applications.
 
-Since object cache stores full objects the cached items are reuseable across multiple pages and work for authenticated traffic - a benefit full page caching doesn't provide.
+For these reasons, persistent object caching support is commonly offered with Managed WordPress hosting.
 
 ![WordPress Object Caching Diagram](/assets/wordpress-object-caching-example.png)
 
 > Transients are inherently sped up by caching plugins, where normal Options are not. A memcached plugin, for example, would make WordPress store transient values in fast memory instead of in the database. For this reason, transients should be used to store any data that is expected to expire, or which can expire at any time. Transients should also never be assumed to be in the database, since they may not be stored there at all.
 
 ###### source: [WordPress Codex](https://codex.wordpress.org/Transients_API)
 
-A good example is a social media widget. A plugin may call out to a third-party API to fetch the latest social content. Rather than doing this on every page load the results can be stored in a transient. WordPress will store transients, these forms of expiring options, in the datbase. In most cases fetching the latest social posts from the database is faster than calling out to the social media API but object caching can improve this further. Rather than store those items in the database WordPress will make use of the persistent object cache, usually a fast memory store.
+**TODO: Consider talking about autoload and Memcached**
```

Comment:
@getsource can we move this to an issue and get rid of inline TODO comments?

Reply:
[Issue & example outline created: #37](https://github.com/ataylorme/WordPress-Hosting-Best-Practices-Documentation/issues/37)